schema.graphql

```diff
@@ -171,15 +171,6 @@ type OwnershipTransferred @entity(immutable: true) {
   transactionHash: Bytes!
 }
 
-type VaultDeleted @entity(immutable: true) {
```

Comment:
This event can't be emitted by the contract, so there's no need to store it

https://github.com/The-Poolz/VaultManager/issues/64